cohttp-curl-lwt/bin/curl.ml

```diff
@@ -0,0 +1,96 @@
+open Lwt.Syntax
+open Cohttp
+module Curl = Cohttp_curl_lwt
+module Sexp = Sexplib0.Sexp
+
+let src =
+  Logs.Src.create "cohttp.lwt.curl" ~doc:"Cohttp Lwt curl implementation"
+
+module Log = (val Logs.src_log src : Logs.LOG)
+
+let client uri ofile meth' =
+  Log.debug (fun d -> d "Client with URI %s" (Uri.to_string uri));
```

Comment:
Maybe it complains about the use of uri because it is needed here

Reply:
This is a binary that isn't public though. It's just there for easy testing.